Fix library name: libsqlite3.so => libsqlite3.so.0
--- a/db_sqlite.c
+++ b/db_sqlite.c
@@ -56,16 +56,16 @@
 
 static void db_sqlite3_dlopen(void)
 {
-	csync_debug(2, "Opening shared library libsqlite3.so\n");
+	csync_debug(2, "Opening shared library libsqlite3.so.0\n");
 
-	dl_handle = dlopen("libsqlite3.so", RTLD_LAZY);
+	dl_handle = dlopen("libsqlite3.so.0", RTLD_LAZY);
 	if (dl_handle == NULL) {
 		csync_fatal
-		    ("Could not open libsqlite3.so: %s\n"
+		    ("Could not open libsqlite3.so.0: %s\n"
 		     "Please install sqlite3 client library (libsqlite3) or use other database (postgres, mysql)\n",
 		     dlerror());
 	}
-	csync_debug(2, "Reading symbols from shared library libsqlite3.so\n");
+	csync_debug(2, "Reading symbols from shared library libsqlite3.so.0\n");
 
 	LOOKUP_SYMBOL(dl_handle, sqlite3_open);
 	LOOKUP_SYMBOL(dl_handle, sqlite3_close);
